Sources/Commands/SwiftTestCommand.swift

```diff
@@ -814,7 +819,7 @@ final class TestRunner {
         #if os(macOS)
         if library == .xctest {
             guard let xctestPath = self.toolchain.xctestPath else {
-                throw TestError.xctestNotAvailable
+                throw TestError.xctestNotAvailable()
```

Comment:
Why not provide a reason here for users to have an actionable error message?

Reply:
Done.